tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 envlist =
-    py{37,38,39,310}-dj32-{sqlite3,postgres,mysql,mariadb},
-    py{38,39,310}-dj{40,41,main}-{sqlite3,postgres,mysql,mariadb},
+    py{37,38,39,310,311}-dj32-{sqlite3,postgres,mysql,mariadb},
```

Comment:
Seems Python 3.11 support added only for Django 4.1 and will be available in next release
https://docs.djangoproject.com/en/dev/faq/install/